akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/scaladsl/EntityRef.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding.typed.scaladsl
```

Comment:
Could we still put it in a testkit named package though?

`akka.cluster.sharding.typed.testkit.{javadsl,scaladsl}`?

Reply:
+1

Reply:
sure